constraints.md

```diff
@@ -1,192 +1,198 @@
----
-title: Constraints
-summary: Learn how SQL Constraints apply to TiDB.
-category: reference
-aliases: ['/docs/dev/reference/sql/constraints/']
----
+# Constraint
 
-# Constraints
+TiDB supports the same constraint as MySQL.
 
```

Comment:
This is not actually the case. TiDB does not support DML for foreign keys, or check constraints.

Reply:
@nullnotnil This has been addressed in L275. Do you think we need to emphasize it in the beginning?

Reply:
Yes, it is better to add some like 'almost all' or something else.

Reply:
> Yes, it is better to add some like 'almost all' or something else.

@KASSADAR Please address this comment.

Reply:
Updated.